data/mods/ssb/statuses.ts

```diff
@@ -38,6 +38,19 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 	IMPORTANT: Obtain the username from getName
 	*/
 	// Please keep statuses organized alphabetically based on staff member name!
+	elgino: {
+		noCopy: true,
+		onStart(target, pokemon) {
+			this.add('-start', pokemon, 'typechange', pokemon.getTypes(true).join('/'), '[silent]');
```

Comment:
We'd want to announce to announce the type change, because people might not notice. Also dont be afraid to hardcode shit. And preferably have the join phrase first

Reply:
![image](https://user-images.githubusercontent.com/19758381/83938140-b2fa6c80-a7ef-11ea-8602-0df362716185.png)
I am sure people will notice this